### DISCUSSION TITLE
A link in the official docs was gone

### DISCUSSION THREAD

**Discussion by BlockyDeer** [RikkaApps/Shizuku#327] [General]
The link is on this page: [here](https://shizuku.rikka.app/introduction/)
![Screenshot_20230801_093542](https://github.com/RikkaApps/Shizuku/assets/102347931/1d0e42ca-9b67-48f3-bda3-820ac676edd7)
[The link](https://www.pixiv.net/en/artworks/75508584)  that displayed as "a charactor" in the doc is a pixiv link and I clicked that, only to find "This work has been deleted or the content ID doesn't exist."

**Comment by alaaisgreat** [comment]
Hi
Is there anyway to start shizuko automatically when i connect my phone to the pc?
..
Im using it xiamoi redmi note 11 pro. Android 11
